src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/services/RadicleProjectApi.java

```diff
@@ -108,7 +109,7 @@ public List<RadPatch> fetchPatches(String projectId, GitRepository repo) {
             return List.of();
         }
         var node = getSeedNode();
-        var url = node.url + "/api/v1/projects/" + projectId + "/patches";
+        var url = node.url + "/api/v1/projects/" + projectId + "/patches?page=0&perPage=" + ALL_IN_ONE_PAGE;
```

Comment:
This API request will fetch only the patches with the state OPEN. So in order to get all the patches you have to make two more requests with the state draft, archived. You can check the available states from here https://github.com/cytechmobile/radicle-jetbrains-plugin/blob/2c989d04977645dbec2b525a6343201900aac315/src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/models/RadPatch.java#L78

Reply:
yes, but this is not really related to the issue I'm trying to resolve, right?
I simply copied the per page option to be similar to issues :)